--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -11,12 +11,6 @@ include_directories(${QT_INCLUDES} ${KDE
 find_package(Alsa)
 alsa_configure_file(${CMAKE_CURRENT_BINARY_DIR}/config-alsa.h)
 
-
-# If these two fail, behavior falls back to pre-git-migration
-# FIXME: After the git migration, we need the REQUIRED param
-find_package(Libkcddb)
-find_package(Libkcompactdisc)
-
 find_package(MusicBrainz3 REQUIRED)
 
 include_directories(
@@ -121,8 +115,7 @@ target_link_libraries(kscd
     ${KDE4_SOLID_LIBS}
     ${KDE4_PHONON_LIBS}
     ${MUSICBRAINZ3_LIBRARIES}
-    ${LIBKCDDB_LIBRARIES}
-    ${LIBKCOMPACTDISC_LIBRARIES})
+    )
 
 ########### install files ###############
 
--- a/kscd.kcfg
+++ b/kscd.kcfg
@@ -3,8 +3,6 @@
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       xsi:schemaLocation="http://www.kde.org/standards/kcfg/1.0
       http://www.kde.org/standards/kcfg/1.0/kcfg.xsd" >
-    <!--<include>KGlobalSettings</include>-->
-    <!--<include>kcompactdisc.h</include>-->
 	<include>KStandardDirs</include>
     <kcfgfile name="kscdrc"/>
     <group name="General">
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -20,8 +20,6 @@ target_link_libraries(HWControler_test
 	${KDE4_PHONON_LIBS}
 	${KDE4_KDECORE_LIBS}
 	${QT_QTTEST_LIBRARY}
-	${LIBKCDDB_LIBRARIES}
-	${LIBKCOMPACTDISC_LIBRARIES}
 )
 
 
@@ -31,7 +29,5 @@ target_link_libraries(AudioCD_test
 	${KDE4_PHONON_LIBS}
 	${KDE4_KDECORE_LIBS}
 	${QT_QTTEST_LIBRARY}
-	${LIBKCDDB_LIBRARIES}
-	${LIBKCOMPACTDISC_LIBRARIES}
 )
 
